scripts/check_format.sh

```diff
@@ -30,7 +30,7 @@ function check_shogun_style {
         echo "-----"
         echo "Running clang-format-3.8 against branch ${2:-}, with hash $BASE_COMMIT"
 
-        COMMIT_FILES=$(git diff --name-only $BASE_COMMIT)
+        COMMIT_FILES=$(git diff --name-only $BASE_COMMIT src/shogun)
```

Comment:
Hi @geektoni 
Is there an appropriate way to fix [this error](https://travis-ci.org/shogun-toolbox/shogun/jobs/254065812#L733)?
Or I'll probably try to change the target path to `src/shogun` as I did here.